asf_search/download/download.py

```diff
@@ -74,14 +73,21 @@ def download_url(url: str, path: str, filename: str = None, session: ASFSession
         for chunk in response.iter_content(chunk_size=8192):
             f.write(chunk)
 
-def remotezip(url: str, session: ASFSession) -> RemoteZip:
+def remotezip(url: str, session: ASFSession):
     """
     :param url: the url to the zip product
     :param session: the authenticated ASFSession to read and download from the zip file
     """
-
-    session.hooks['response'].append(strip_auth_if_aws)
-    return RemoteZip(url, session=session)
+    try:
+        import remotezip
+    except ImportError:
+        remotezip = None
+    
+    if remotezip is None:
+        ASF_LOGGER.warn("Could not find remotezip package in current python environment. \"remotezip\" is an optional dependency of asf-search required for the `remotezip()` method, and can be install via `python3 -m pip install asf-search[extras]`")
+    else:
+        session.hooks['response'].append(strip_auth_if_aws)
+        return remotezip.RemoteZip(url, session=session)
```

Comment:
```suggestion
    
    session.hooks['response'].append(strip_auth_if_aws)
    return remotezip.RemoteZip(url, session=session)
```

Totally stylistic and varies from team to team, but I prefer not to have hanging else blocks at the end of functions as this clearly demarcates the happy-path and end of the function